Cannot compile OF0.9.3 with latest MSYS2

Compilation fails due to missing definition of DEVPROPKEY and DEVPROPTYPE in setupapi.h.

```
In file included from C:/Users/Public/Msys4Of/openFrameworks/libs/openFrameworks/communication/ofSerial.h:15:0,
                 from C:/Users/Public/Msys4Of/openFrameworks/libs/openFrameworks/ofMain.h:38,
                 from C:/Users/Public/Msys4Of/openFrameworks/libs/openFrameworks/video/ofDirectShowPlayer.h:6,
                 from C:/Users/Public/Msys4Of/openFrameworks/libs/openFrameworks/video/ofDirectShowPlayer.cpp:1:
C:/Users/Public/Msys4Of/msys64/mingw32/i686-w64-mingw32/include/setupapi.h:1879:119: error: 'DEVPROPKEY' does not name a type
   WINSETUPAPI WINBOOL WINAPI SetupDiGetDevicePropertyW(HDEVINFO DeviceInfoSet, PSP_DEVINFO_DATA DeviceInfoData, const DEVPROPKEY *PropertyKey, DEVPROPTYPE *PropertyType, PBYTE PropertyBuffer, DWORD PropertyBufferSize, PDWORD RequiredSize, DWORD Flags);
                                                                                                                       ^~~~~~~~~~
C:/Users/Public/Msys4Of/msys64/mingw32/i686-w64-mingw32/include/setupapi.h:1879:144: error: 'DEVPROPTYPE' has not been declared
   WINSETUPAPI WINBOOL WINAPI SetupDiGetDevicePropertyW(HDEVINFO DeviceInfoSet, PSP_DEVINFO_DATA DeviceInfoData, const DEVPROPKEY *PropertyKey, DEVPROPTYPE *PropertyType, PBYTE PropertyBuffer, DWORD PropertyBufferSize, PDWORD RequiredSize, DWORD Flags);
                                                                                                                                                ^~~~~~~~~~~
C:/Users/Public/Msys4Of/msys64/mingw32/i686-w64-mingw32/include/setupapi.h:1880:116: error: 'DEVPROPKEY' has not been declared
   WINSETUPAPI WINBOOL WINAPI SetupDiGetDevicePropertyKeys(HDEVINFO DeviceInfoSet, PSP_DEVINFO_DATA DeviceInfoData, DEVPROPKEY *PropertyKeyArray, DWORD PropertyKeyCount, PDWORD RequiredPropertyKeyCount, DWORD Flags);
                                                                                                                    ^~~~~~~~~~
```

Adding a `#include <devpropdef.h>` just before `#include <setupapi.h>` in ofSerial.h seems to solve the problem.

--- a/libs/openFrameworks/communication/ofSerial.h
+++ b/libs/openFrameworks/communication/ofSerial.h
@@ -11,6 +11,7 @@
 	#include <tchar.h>
 	#include <iostream>
 	#include <string.h>
+	#include <devpropdef.h>
 	#include <setupapi.h>
 	#include <regstr.h>
 	/// \cond INTERNAL

